src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -1489,6 +1454,79 @@ public static function debug_data() {
 		return $info;
 	}
 
+	/**
+	 * Populate the file system section of the debug data.
+	 *
+	 * Using the {@see 'debug_information'} filter for this.
```

Comment:
```suggestion
	 * Gets the filesystem section of the debug data.
	 *
```
This could just be a getter method, called in `::debug_data()` something like I mentioned above.

Reply:
Will change it but rephrase your suggestion to be conform with WPCS.